app/views/manuals/show.html+new_navigation.erb

```diff
@@ -14,12 +14,19 @@
 
 <div class='grid-row'>
   <div class='column-two-thirds'>
```

Comment:
If you applied a `new-navigation` class on this wrapper `div`, instead of in the partials, you could avoid passing this flag around and doing all this checks in the partials, and move the logic into CSS (possibly where it belongs). It also helps to alleviate the fact that passing a flag into these partials is a bit inconsistent with the use of a variant elsewhere.